rotation/node.go

```diff
@@ -6,7 +6,7 @@ package rotation
 import (
 	"context"
 	"fmt"
-
+	"github.com/golang/protobuf/proto"
```

Comment:
This is the old protobuf version module import, we'll want
```suggestion
	"google.golang.org/protobuf/proto"
```

Reply:
ty!